tracim/tests/library/test_user_api.py

```diff
@@ -60,26 +63,84 @@ def test_get_one_by_email_exception(self):
         with pytest.raises(NoResultFound):
             api.get_one_by_email('unknown')
 
-    def test_get_all(self):
-        # TODO - G.M - 29-03-2018 Check why this method is not enabled
+    # def test_unit__get_all__ok__nominal_case(self):
+    #     # TODO - G.M - 29-03-2018 Check why this method is not enabled
+    #     api = UserApi(
+    #         current_user=None,
+    #         session=self.session,
+    #         config=self.config,
+    #     )
+    #     u1 = api.create_user(True)
+    #     u2 = api.create_user(True)
+    #     users = api.get_all()
+    #     assert 2==len(users)
+
+    def test_unit__get_one__ok__nominal_case(self):
         api = UserApi(
             current_user=None,
             session=self.session,
             config=self.config,
         )
-        # u1 = api.create_user(True)
-        # u2 = api.create_user(True)
+        u = api.create_user()
+        api.update(u, 'titi', 'titi@titi', True)
+        one = api.get_one(u.user_id)
+        eq_(u.user_id, one.user_id)
 
-        # users = api.get_all()
-        # ok_(2==len(users))
+    def test_unit__get_user_with_context__nominal_case(self):
+        user = User(email='admin@tracim.tracim')
+        api = UserApi(
+            current_user=None,
+            session=self.session,
+            config=self.config,
+        )
+        new_user = api.get_user_with_context(user)
+        assert isinstance(new_user, UserInContext)
+        assert new_user.user == user
```

Comment:
you should also check that the context is valid (eg: user avatar url, ...)

Reply:
Done.